mathics/doc/common_doc.py

```diff
@@ -769,27 +780,30 @@ def get_tests(self) -> Iterator[Tests]:
                     index = 1
                     for doctest in section.items:
                         if isinstance(doctest, DocTests):
-                            doctest.index = index
-                            doctest.chapter = self.chapter.title
-                            doctest.part = self.chapter.part.title
-                            doctest.section = self.title
-                            doctest.key = (
-                                doctest.part,
-                                doctest.chapter,
-                                doctest.section,
+                            # Why monkey patch DocTests?
+                            # These attributes corresponds to a `Tests`
+                            # object.
+                            doctest_index = index
+                            doctest_chapter = self.chapter.title
+                            doctest_part = self.chapter.part.title
+                            doctest_section = self.title
+                            doctest_key = (
+                                doctest_part,
+                                doctest_chapter,
+                                doctest_section,
                                 index,
                             )
                             index += 1
                             new_items.append(doctest)
-
                             if len(new_items) > 0:
                                 self.tests = Tests(
-                                    doctest.part,
-                                    doctest.chapter,
-                                    doctest.section,
+                                    doctest_part,
+                                    doctest_chapter,
+                                    doctest_section,
                                     new_items,
                                 )
                                 yield self.tests
+                                1 / 0
```

Comment:
What's this line for?

Reply:
This is just a mistake. I put it to see if the function was called in some place, and I forgot to remove it.